substrate/client/network/sync/src/engine.rs

```diff
@@ -741,7 +738,25 @@ where
 			ChainSyncAction::CancelBlockRequest { peer_id } => {
 				let removed = self.pending_responses.remove(&peer_id);
 
-				trace!(target: LOG_TARGET, "Processed {action:?}., response removed: {removed}.");
+				trace!(target: LOG_TARGET, "Processed {action:?}, response removed: {removed}.");
+			},
+			ChainSyncAction::SendStateRequest { peer_id, request } => {
+				self.send_state_request(peer_id, request);
+
+				trace!(
+					target: LOG_TARGET,
+					"Processed `ChainSyncAction::SendBlockRequest` to {peer_id}.",
+				);
+			},
+			ChainSyncAction::SendWarpProofRequest { peer_id, request } => {
+				self.send_warp_proof_request(peer_id, request.clone());
+
+				trace!(
+					target: LOG_TARGET,
+					"Processed `ChainSyncAction::SendWarpProofRequest` to {}, request: {:?}.",
+					peer_id,
+					request,
```

Comment:
```suggestion
					"Processed `ChainSyncAction::SendWarpProofRequest` to {peer_id}, request: {request:?}.",
```

Reply:
We don't fit here into 100 character per line. It's the only reason why this is not inlined.